post_office/templates/admin/post_office/email/change_form.html

```diff
@@ -0,0 +1,5 @@
+{% extends "admin/change_form.html" %}{% load i18n %}
+{% block object-tools-items %}
+    {{ block.super }}
+    <li><a href="{% url 'admin:resend' original.pk %}" style="background-color:#0099ff">{% trans "Resend" %}</a></li>
```

Comment:
@selwin to make it a POST, I would need to make a form here or you do have another idea

Reply:
<form method="post" url=reverse url>
  <button type="submit">
  </button>
</form>

Reply:
Yes, we should make a django admin button. It should be something like this https://stackoverflow.com/a/45072862

Reply:
it's still a get since is using <a href..
it's already a button tho

Reply:
<img width="1410" alt="Screen Shot 2021-12-12 at 4 34 38 PM" src="https://user-images.githubusercontent.com/2621572/145732229-25031d1e-ff5b-449f-bd1a-2b26f848cdd6.png">
this is the look and feel

Reply:
@selwin do you think this is ok?

Reply:
I'm ok with the look and feel. If you can make it a POST request, it would be great because it's more technically correct.

Reply:
> to make it a POST, I would need to make a form here or you do have another idea

Yes, I think this is the right solution solution.